### DISCUSSION TITLE
Dudas con la practica 1

### DISCUSSION THREAD

**Discussion by DaveOval** [Launch-X-Latam/MisionFrontEnd#120] [Q&A]
Hola, perdón por la pregunta pero realmente estoy perdido, no entiendo que tenemos que hacer en la practica 1.
- A lo que yo entendí tenemos que hacer un Wireframe en donde se muestre como será la pagina que nos solicita el cliente sin tanto detalle mostrando como sera la interaccion en la pagina.
- Y despues hacer en figma o plataforma a eleccion el UI (Que es con colores y toda la cosa)
- pero no entiendo la parte de toma de requerimento si tenemos que llenar el word que nos dan para descargar o que se tiene que hacer con eso
- y por ultimo en que formato lo entregaremos (Creo que subiendolo a github pero no estoy seguro el como se tiene que subir)
Muchas gracias por responder y espero que me ayuden a comprender la actividad.

**Comment by samujm** [comment]
Hola, mira, el desarrollo de un sistema web, una página de ventas, un sistema de escritorio o una app móvil siempre va acompañado de todos los pasos que nos fueron indicando en las clases, al menos así debería ser.

Una de las partes más importantes del desarrollo es el análisis, podemos decir que esos son los cimientos para que un desarrollo no se nos caiga, por decirlo de manera coloquial.
El documento que entregan es algo formal que se le entrega al cliente de conformidad como si tu fueras tu una empresa de desarrollo (o si estas en una así debe ser), es la manera correcta para delimitar las cosas que quiere el cliente, te pongo un ejemplo: yo soy la persona que atiende al cliente y me toca recabar todos los requerimientos del mismo, lo hablamos y me dice que él quiere un botón en la esquina superior derecha de color rojo, por decir algo, supongamos que ese es su único requisito, en una segunda visita le mostramos el boceto en figma (o en cualquier software), y él me va a decir que el botón que el pidió era verde, no rojo. Si yo no realizo un documento firmado tanto por el cliente y por mi dónde los dos estamos en la misma sintonía de sus peticiones él va a poder hacer y deshacer a su antojo. Por ello se crea ese documento.

Regresando a tu pregunta, yo creo que lo más conveniente es llenar las partes del documento que si se puedan llenar, ignoro si van a enseñar un poco de los diagramas de gantt, pero bueno, te recomiendo analizar bien el diagrama de actividades, crear los Wiframe o la idea a grandes rasgos de los requerimientos que nos están dando, y por ultimo crear el UI donde te puedes súper extender y hacer volar tu imaginación.

En algún lugar vi que alguien comentó que todo eso se sube a repositorio y al finalizar el curso de front solicitan todo, pero la verdad no estoy segura. Espero te pueda ayudar mi larga explicación y cualquier duda no dudes en comentarme. Saludos.

**Comment by Amhedriel** [comment]
Buenas Noches Dave.

La primera parte de tu pregunta fue contestada en otra conversación, pero en resumen un compañero dejo un ejemplo en su [GitHub](https://github.com/JosafatJimenezB/MissionFrontEnd "Práctica 1"), me parece que estuvo en un programa similar asi que es un buen ejemplo.

Creo, no estoy seguro, si debemos subir a GitHub como el colega del enlace anterior porque el buen Sherpa no nos dijo que hacer para subir la práctica, nos dijo que vayamos con "calma" que él nos guiará por el proceso, así que revisa los videos de los Live.

Por mi parte, tengo un pequeño tutorial para Git y algo de GitHub que esta en [MarkDown](https://github.com/Amhedriel/Mark_Down_Tutoriales "Bienvenido") (que también tiene una guía rápida), que es el formato que se utiliza en GitHub como el usado en estos textos y cualquier duda me la puedes enviar.

Saludos.
